ipatests/prci_definitions/nightly_master_pki.yaml

```diff
@@ -782,3 +782,15 @@ jobs:
         template: *pki-master-f30
         timeout: 3600
         topology: *master_1repl
+
+  fedora-30/test_ca_custom_sdn:
+    requires: [fedora-30/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-30/build_url}'
+        test_suite: test_integration/test_ca_custom_sdn.py
```

Comment:
We need to add `update_packages: True` in nightly_master_pki.yaml, so that the latest build of PKI is installed.